simplq/src/store/asyncActions/getQueueStatusByName.js

```diff
@@ -0,0 +1,41 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+import { useHistory } from 'react-router-dom';
+import useAuth, { makeAuthedRequest } from 'api/auth';
+import * as RequestFactory from 'api/requestFactory';
+
+const typePrefix = 'getQueueStatusByName/requestStatus';
+
+/**
+ * A hook to access the getQueueStatusByName async action creator.
+ *
+ * @returns — getQueueStatusByName async action creator
+ */
+const useGetQueueStatusByName = () => {
+  const auth = useAuth();
+  const history = useHistory();
+
+  const getQueueStatusByName = createAsyncThunk(
+    typePrefix,
+    async ({ queueName }, { rejectWithValue }) => {
+      if (!auth || !auth.isAuthenticated) {
+        return {};
+      }
```

Comment:
Here we should check just for `auth.isLoading`, as the queue status endpoint doesn't need authentication.